### PR TITLE
Features/log

### PR DESCRIPTION
**Pull request by Stalfy**
Loggers

### CODE REVIEW

**Review by doomy23 — APPROVED** [review]
Cool stuff !

**Review by KB07 — APPROVED** [review]
Good to me.

File: src/parser/MarkdownParser.js [hunk 1]
```diff
@@ -27,11 +27,10 @@ class MarkdownParser {
         this.params = {};
     }
 
-    parseFile(filePath, options){ // eslint-disable-line no-unused-vars
+    parseFile(filePath) {
```
Comment (Grasseh) [inline src/parser/MarkdownParser.js:30]: At this point maybe you want to add this :
https://eslint.org/docs/rules/space-before-blocks

To the eslintrc ^^

File: src/util/ConfigParser.js [hunk 2]
```diff
@@ -2,9 +2,9 @@ const cosmiconfig = require('cosmiconfig');
 const ArgumentParser = require('argparse').ArgumentParser;
 
 class ConfigParser {
-    parseConfig() {
+    parseConfig(logger) {
         let args = this.parseArgs();
-        return { paths: this.parsePaths(args.config), args };
+        return { paths: this.parsePaths(args.config, logger), args };
```
Comment (Grasseh) [inline src/util/ConfigParser.js:7]: Would it be better to set the logger as an attribute of the class instead of passing it as param?

Up to debate tbh

**Review by Grasseh — APPROVED** [review]
Two points of interest for maaaaybe in the future, rest is top notch.

**Pull request merged**